utils/nn-parser.js

```diff
@@ -12,6 +12,9 @@ var findInitParams = function(ast){
     ast.block.stats.forEach(function(block){
         if(block.key && block.key.val == '__init' && block.func){
             params = block.func.args;
+            if(params.length == 0 && block.func.varargs){
+               params[0] = 'VariableArguments'
```

Comment:
Whatever name is provided here will be used in the web interface. Would something like `params` be a better fit?